.github/workflows/CI.yml

```diff
@@ -73,6 +73,9 @@ jobs:
     runs-on: macos-11
     needs: [generateNativeConfig]
     steps:
+      - uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: '12.5.1'
       - uses: actions/checkout@v2
```

Comment:
https://github.com/actions/runner-images/blob/main/images/macos/macos-11-Readme.md#xcode
xcode 12.5.1 is available for mac11